demos/include/aws_clientcredential_keys.h

```diff
@@ -65,6 +65,8 @@
  *
  * @todo If you are running one of the FreeRTOS demo projects, set this
  * to the private key that will be used for TLS client authentication.
+ * Please note that this is for demo purposes ONLY and NOT for production
+ * scenarios. NEVER use a genuine secure key like this.
```

Comment:
Please note that pasting a key into header files is for demonstration purpose ONLY and not for production scenarios. NEVER use a genuine security key in a header file in this manner.

Reply:
update the text